client/src/components/BasicTable.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import './table.css';
+import '../App'
+
+
+const RowDetails = () => {
+  return (
+    <p>
+      Some Details
+    </p>
+  )
+}
+
+export const TableRow = (props) => {
+  const [showDetails, setShowDetails] = useState(false);
+
+  const handleRowClick = () => {
+    showDetails ? setShowDetails(!showDetails) : setShowDetails(!showDetails)
```

Comment:
This doesn't make sense, you call setShowDetails(!showDetails) no matter what condition is